thinkhazard/static/less/common.less

```diff
@@ -37,6 +37,13 @@ ul.horizontal li {
 .navbar {
   border: 0;
   .box-shadow(0px 2px 10px -1px rgba(0, 0, 0, 0.33));
+  li:last-child a{
+      padding-top: 0px;
+      padding-bottom: 0px;
+      margin-top: 29px;
+      margin-bottom: 29px;
+      border-left: 1px solid #d9d9d9;
```

Comment:
Yes, the idea is to have a separate element to show the divider. The divider and the language button should be different elements no matter how each is styled.
Using borders to style the divider isn't a problem.